jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -342,7 +256,8 @@ local patchOrExcludeRule(rule, ruleSet, operation) =
   if std.length(ruleSet) == 0 then
     [rule]
   else if ('alert' in rule) then
-    local matchedRules = std.filter(function(ruleItem) ('alert' in ruleItem) && (ruleItem.alert == rule.alert), ruleSet);
+    // empty alert name is matching-all
```

Comment:
we shouldn't need to update the `patchOrExcludeRule` function if we only stick to updating the severities so would you mind reverted the changes you made to it in https://github.com/openshift/cluster-monitoring-operator/pull/1310?

Reply:
of course, this will be restored to previous version

Reply:
restored now :)